pyproject.toml

```diff
@@ -94,6 +95,7 @@ all = [
     "nvidia-modulus[dev]",
     "nvidia-modulus[makani]",
     "nvidia-modulus[fignet]",
+    "multi-storage-client[boto3]",
```

Comment:
Why only install boto3 and not other backends in the `all` dep list?

Reply:
Thanks for pointing out! I removed the boto3 here as it should be installed in the example folder instead.